source/includes/steps-create-data-encryption-key.yaml

```diff
@@ -0,0 +1,120 @@
+title: Read the local master key from a file
+ref: read-local-master-key-from-file
+level: 4
+content: |
+  First, read the file containing the local master key, generated in the
+  :ref:`Create a Master Key <fle-create-a-master-key>` section:
+
+  .. tabs::
+
+     tabs:
+
+       - id: use-cases-fle-guide-java
+         name: "Java"
+         content: |
+
+           .. code-block:: java
+              :emphasize-lines: 11
+
+              String path = "master-key.txt";
+
+              byte[] fileBytes = new byte[96];
+
+              try (FileInputStream fis = new FileInputStream(path)) {
+                fileBytes = fis.readAllBytes();
+              } catch (IOException e) {
+                e.printStackTrace();
+              }
+
+              final byte[] localMasterKey = Arrays.copyOf(fileBytes, 96);
+
+
+---
+title: Specify KMS provider settings
+ref: specify-kms-provider-settings
+level: 4
+content: |
+  Next, specify the KMS provider settings. Set the provider name to ``local``
+  when using a local master key:
```

Comment:
I think we could have a little more detail on the what the KMS provider settings actually are / what they configure.

Reply:
I was planning to mention the AWS KMS-specific ones in the final section of the use-case guide, but are there other settings and are they important to mention?

Reply:
Don't think there's any settings we don't have in this or the final step. I'm thinking more of "the client uses this settings map to access the master key"

Reply:
Makes sense, will add detail.